xcodeproj/internal/input_files.bzl

```diff
@@ -321,7 +321,10 @@ https://github.com/buildbuddy-io/rules_xcodeproj/issues/new?template=bug.md
             generated,
             transitive = [
                 info.inputs.generated
-                for _, info in transitive_infos
+                for attr, info in transitive_infos
+                if (not attrs_info or
+                    info.target_type in
+                    attrs_info.xcode_targets.get(attr, [None]))
```

Comment:
Curious. Why `[None]` vs `[]`?

Reply:
Because it does an `in` check, and `info.target_type` can be `None`, and that needs to be `True`.